.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Done
+
+* [List of work items including drive-bys]
+
+## QA
+
+- Either use the demo server
+  - via the link below to demos.haus
+- Or run the project locally
+  - as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development)
+- [List additional steps to QA the new features or prove the bug has been resolved]
```

Comment:
```suggestion
Perform the following QA steps with the demo server (for PRs created by collaborators of the repo) or with a local clone of this branch:
- Open [demo](insert-demo-url)
- [Add QA steps]
```

Reply:
The demo link is just pasted after the PR is created, and webbot will comment it right after the description. I'd keep it with the hint to the link.

Reply:
Sure, you can keep the hint to the link, but the text you proposed was not very clear and also doesn't highlight the fact that @webteam-app will only create demos for PRs submitted by collaborators of the repo. Since we have also community contributors, and since this template is meant to aid them too with the description of the QA steps, I'd make this very clear.